SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the latest release.
```

Comment:
This is true, that we will only apply the security updates when a new release comes out. However, we will also provide patches for security bugs on our project website (https://tukaani.org/xz), so this should be mentioned here. We will provide patches for all supported development branches, which will always be at least the current development branch and the current stable branch. Right now we still maintain an old stable branch, but that might not always be the case.